tests/ocp/sriov/internal/ocpsriovconfig/config.go

```diff
@@ -117,3 +218,26 @@ func readEnv(sriovOcpConfig *SriovOcpConfig) error {
 
 	return nil
 }
+
+// parseLegacyVFNum parses the legacy SRIOV_VF_NUM environment variable.
+// Only applies if ECO_OCP_SRIOV_VF_NUM was not set.
+func parseLegacyVFNum(sriovOcpConfig *SriovOcpConfig) {
```

Comment:
Why do we need this func? Why  are you going to continue using SRIOV_VF_NUM instead of ECO_OCP_SRIOV_VF_NUM?